code/drasil-docLang/Drasil/Sections/TableOfAbbAndAcronyms.hs

```diff
@@ -25,7 +25,7 @@ select (x:xs) = case getA x of
 -- | The actual table creation function.
 table :: (Idea s) => [s] -> Contents
 table ls = let chunks = sortBy (compare `on` fst) $ select ls in
-  Table (map (at_start) [symbol_, description]) (mkTable
+  Table [S "Abbreviation", S "Full Form"] (mkTable
```

Comment:
ditto

Reply:
So should `abbreviation` and  `fullForm` be added to Data.Drasil.Concepts.Documentation as `NamedChunk`s?

Eg:
https://github.com/JacquesCarette/Drasil/blob/2dbd9375c89c7c73c3a88a1692fc714f3962e94f/code/drasil-data/Data/Drasil/Concepts/Documentation.hs#L58
https://github.com/JacquesCarette/Drasil/blob/2dbd9375c89c7c73c3a88a1692fc714f3962e94f/code/drasil-data/Data/Drasil/Concepts/Documentation.hs#L212

Reply:
Sounds like a good idea.